### PR TITLE
Fix Minor Misspelling in run a node developers doc (Russian)

### PR DESCRIPTION
**Pull request by shamshod01**
## Description 
Minor misspeling in russian word

## Related Issue
missing letter O in word  локальных

### CODE REVIEW

**Review by minimalsm — COMMENTED** [review]
Thanks @shamshod01, appreciate the PR 💪 

I've pinged some Russian speakers to get a second opinion before merging this.

**Review by corwintines — APPROVED** [review]
Thanks, @shamshod01! Has this been updates in crowdin as well?

**Pull request merged**
